docs/designers-developers/developers/block-api/block-supports.md

```diff
@@ -194,6 +194,45 @@ supports: {
 }
 ```
 
+## duotone
+
+- Type: `boolean`, `string`, `string[]`, or `object`
+- Default value: `false`
+
+This property adds block controls which allow to apply a duotone filter to a block or part of a block. _Important: It doesn't work with dynamic blocks yet._
+
+```js
+supports: {
+    // Declare support for duotone.
+    // This applies the duotone filter to the entire block in both edit and save.
+    duotone: true
+}
+```
+
+```js
+supports: {
+    // Apply the filter to the same selector in both edit and save.
+    duotone: 'img'
```

Comment:
@nosolosw is also adding a "selector" support  for some other block supports separately, we should make sure these APIs are similar.